<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficLightView</class>
 <widget class="QMainWindow" name="TrafficLightView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Light Simulation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
background-color: rgb(0, 0, 0);
border: transparent;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTraffic_Light">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuItemAbout"/>
    <addaction name="separator"/>
    <addaction name="menuItemExit"/>
   </widget>
   <addaction name="menuTraffic_Light"/>
  </widget>
  <action name="menuItemExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menuItemAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
